kong/meta.lua

```diff
@@ -18,7 +18,7 @@ return {
   -- third-party dependencies' required version, as they would be specified
   -- to lua-version's `set()` in the form {from, to}
   _DEPENDENCIES = {
-    nginx = {"1.11.2.1"},
+    nginx = {"1.11.2.1", "1.11.2.2"},
```

Comment:
1.11.2.2 is the first one to include the breaking Lua 5.2 features, can't have this without pathcing `execute` in Penlight